src/core/scenario.jl

```diff
@@ -221,57 +221,57 @@ function Scenario(d::Dict; flex_hvac_from_json=false)
     if haskey(d, "DomesticHotWaterLoad") && !haskey(d, "FlexibleHVAC")
         add_doe_reference_names_from_elec_to_thermal_loads(d["ElectricLoad"], d["DomesticHotWaterLoad"])
         existing_boiler_efficiency = get_existing_boiler_efficiency(d)
-        year = get(d["DomesticHotWaterLoad"], "year", electric_load.year)
+        year = get!(d["DomesticHotWaterLoad"], "year", electric_load.year)
         validate_load_year_consistency(electric_load.year, year, "DomesticHotWaterLoad")
         dhw_load = HeatingLoad(; dictkeys_tosymbols(d["DomesticHotWaterLoad"])...,
                                         load_type = "domestic_hot_water",
                                         latitude=site.latitude, longitude=site.longitude, 
                                         time_steps_per_hour=settings.time_steps_per_hour,
-                                        existing_boiler_efficiency = existing_boiler_efficiency,
-                                        year = year
+                                        existing_boiler_efficiency = existing_boiler_efficiency
                                         )
         max_heat_demand_kw = maximum(dhw_load.loads_kw)
     else
+        # TODO need to add a dummy year here or else will error
```

Comment:
@Bill-Becker is this still a pending TODO?

Reply:
Nope, done.